advisor/src/main/kotlin/advisors/Osv.kt

```diff
@@ -168,19 +171,30 @@ private fun createRequest(pkg: Package): VulnerabilitiesForPackageRequest? {
 }
 
 private fun Vulnerability.toOrtVulnerability(): org.ossreviewtoolkit.model.Vulnerability {
-    val (scoringSystem, severity) = severity.firstOrNull()?.let {
-        it.type.name to it.score
-    } ?: null to null
+    val (scoringSystem, score) = severity.firstOrNull()?.let {
+        val cvss = Cvss.fromVector(it.score) ?: run {
+            log.debug { "Could not parse CVSS vector from: '${it.score}'." }
```

Comment:
This reads as if you're trying to *create* a vector from `it.score`. Just say "Could not parse CVSS vector '${it.score}'."